app/assets/stylesheets/comments.scss

```diff
@@ -686,8 +695,7 @@ a.header-link {
       font-size: 12px;
       text-align: right;
       a {
-        color: $medium-gray;
-        color: var(--theme-secondary-color, $dark-gray);
+        @include themeable(color, theme-secondary-color, $medium-gray);
```

Comment:
I'm gonna change this to `$dark-gray`, since that's what we have as the default value for the CSS variable.

Reply:
Nice catch. The previous code was broken. I guess we'll see if this looks wrong, but I think it's going back to what it was before.